### PR TITLE
Userstory3#29

### LINKED ISSUE #29: User Story 3, All book titles are links to a show page
```
As a visitor
With the exception of a book's show page,
Anywhere I see a book title on the site,
I can click on the book title to go to that book's show page.
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written
- [ ] edge case code written
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #31

child of #33

depends on #20

### PR DESCRIPTION
**Pull request by earl-stephens**
Completed user story 3, turning book titles into hyperlinks to the book's show page.  Tests are passing.  Closes #29

### CODE REVIEW

File: app/views/welcome/index.html.erb [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+  <section class="intro_msg">
+    <h1>Welcome to Book Club!</h1>
+    <p>We have a variety of books for you to browse through.</p>
+    <p>You can sort our books in a variety of ways,<br/>
+      or check out the authors, the reviews, or the reviewers.</p>
+    <p>And feel free to leave your own review!</p>
+    <p>Enjoy!</p>
+  </section>
+</section>
+
```
Comment (djc00p) [inline app/views/welcome/index.html.erb]: this is looking good thanks for getting that done
